README.md

```diff
@@ -47,10 +53,13 @@ Draft requires Go version 1.18.x.
    mv draft $GOPATH/bin/
    ```
 
+## Contributing
+Draft is fully compatible with [Azure Kubernetes Services](https://docs.microsoft.com/en-ca/azure/aks/draft). We strongly encourage contributions to make Draft available to other cloud providers 😊!
+
 ## License
 
-Distributed under the MIT License. See [LICENSE](https://github.com/Azure/draft/blob/main/LICENSE) for more information.
+Draft is under the MIT License. See [LICENSE](https://github.com/Azure/draft/blob/main/LICENSE) for more information.
 
 ## Trademark Notice
```

Comment:
💡☕️ thought share: this is trademark stuff, is this approved by CELA? Also I am not sure how is this getting generated because there is an auto generated one we get when we first setup the repository so I thought I will share this . 🙏☕️